### DISCUSSION TITLE
Getting 4bit LoRA "--monkey-patch" to work will be my undoing. Workarounds?

### DISCUSSION THREAD

**Discussion by mkammes** [oobabooga/text-generation-webui#1665] [Q&A]
I've deployed more WSL and Windows 11 and Google Collab environments with text-generation-webui than I care to admit, and getting monkey-patch to run is proving to be...difficult.

Has anyone found any tricks that have helped?

Thanks!

**Comment by mikolodz** [comment]
Same here on Ubuntu 22.04. No way to have monkey patch running and still get legit responses. Either some gibberish or no response at all (triton / cuda). Using rtx 3090.

**Comment by Zhincore** [comment]
I'm trying to run monkey-patch on Vicuna 13B GPTQ, but she only replies with gibberish both with and without a LoRA.
I don't know if I missed some step or maybe wrong version of some lib, I can't figure it out...

**Reply by mikolodz** [reply]
If it's still the issue, create a fresh installation of the whole setup (text-generation-webui, gptq, alpaca lora 4b). It solved my problem.

Before doing that, try using no-act-order model. For instance, **I can't use wizard with monkey-patch, but I successfully used and tuned wizard-vicuna.**

**Comment by TheNitzel** [comment]
> 

Did you ever find a solution.  I'm just so over it after hours and hours of frustration.
